_build_cfg.yml

```diff
@@ -265,6 +265,10 @@ Topics:
     File: configuring_authentication
   - Name: Syncing Groups With LDAP
     File: syncing_groups_with_ldap
+  - Name: Setting up SSSD for LDAP Failover
+    File: sssd_for_ldap_failover
+  - Name: Configuring Form-Based Authentication
+    File: configuring_form_based_authentication
```

Comment:
Discussed with @adellape and will be putting these 3 files under Advanced LDAP Configuration